FrontEnd/src/components/HeaderFooter/header/menu/HeaderMenu.jsx

```diff
@@ -20,7 +20,7 @@ const SERVICES_LINKS = [
     },
     {
         title: 'Інші послуги',
-        link: '#'
+        link: '/profiles/otherServices'
```

Comment:
It is better to avoid joining the words in URLs and use a hyphen to separate them - other-services instead of otherServices.

https://developers.google.com/search/docs/crawling-indexing/url-structure?hl=en&visit_id=638485234540125074-3362765097&rd=1

Reply:
Thanks for the info.